config/gitleaks-guide.md

```diff
@@ -1,4 +1,4 @@
-# Using gitleaks to check for PII and credentials #
+# Check for PII and Credentials with `gitleaks` #
```

Comment:
```suggestion
# Check for PII and credentials with `gitleaks` #
```

I think this is what we are doing, right?  Also, do we want to expand "PII" (either in the header or in the text below) for those who are unfamiliar with the acronym?